docs/PLUGINS.md

```diff
@@ -27,12 +27,18 @@ containerd allows extensions through two methods:
 
 ### V2 Runtimes
 
-The runtime v2 interface allows resolving runtimes to binaries on the system.
-These binaries are used to start the shim process for containerd and allows
-containerd to manage those containers using the runtime shim api returned by
+containerd supports multiple container runtimes. Each container can be
+invoked with a different runtime. The runtimes can be configured in the
+containerd configuration file, including setting a default runtime,
```

Comment:
Is there any other way? I thought under v2, all of the runtimes are listed solely in that section of the config?

Reply:
containerd itself only has a single hard-coded default per OS: https://github.com/containerd/containerd/blob/b30e0163ac36c1a193604e5eca031053d62019c5/defaults/defaults_linux.go#L21

When a client (such as `ctr` or `nerdctl`) attempts to run a container, it will either use that hard-coded default or specify the shim as an option.

The CRI plugin (which is effectively another client) has its own configuration section which can specify other defaults.

Reply:
pls include Sam's text here "When a client (such as ctr or nerdctl) attempts to run a container, it will either use that hard-coded default or specify the runtime shim as an option."